Challenges

nayanapardhekar opened the pull request:
Challenge 5 Completed 
Created a folder under contributors with nayanapardhekar.md file along with details in it.

shivaylamba committed:
Spashed all commits

Signed-off-by: Nayana Pardhekar <nayanapardhekar@gmail.com>

kuraakhilesh8230 commented:
congrats you have completed the milestone-1 successfully and you are good to go🎉

nayanapardhekar commented:
Thank you..

On Wed, 14 Sep, 2022, 8:09 pm Kura Akhilesh, ***@***.***>
wrote:

> congrats you have completed the milestone-1 successfully and you are good
> to go🎉
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/678#issuecomment-1246869472>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVZCW3NWHMIN2CJ5IGYYKX3V6HPTNANCNFSM6AAAAAAQFA3GSQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <scaleracademy/scaler-september-open-source-challenge/pull/678/c1246869472
> @github.com>
>

nayanapardhekar committed:
Create HappyNumber.cpp

This file contains the c program for Happy Number
A number is said to be happy if it yields 1 when replaced by the sum of squares of its digits repeatedly. If this process results in an endless cycle of numbers containing 4, then the number will be an unhappy number.

Signed-off-by: Nayana Pardhekar <nayanapardhekar@gmail.com>

nayanapardhekar committed:
Update and rename HappyNumber.cpp to HappyNumber.c

Signed-off-by: Nayana Pardhekar <nayanapardhekar@gmail.com>

kuraakhilesh8230 commented:
too many commits do correct it asap

kuraakhilesh8230 commented:
All checks failed (dco+lint)
Correct it asap

nayanapardhekar commented:
May I check it tomorrow please.

On Tue, 27 Sep, 2022, 11:33 pm Kura Akhilesh, ***@***.***>
wrote:

> All checks failed (dco+lint)
> Correct it asap
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/678#issuecomment-1259867956>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVZCW3NNFGZIHTJZLFQDGXTWAMZITANCNFSM6AAAAAAQFA3GSQ>
> .
> You are receiving this because you modified the open/close state.Message
> ID:
> <scaleracademy/scaler-september-open-source-challenge/pull/678/c1259867956
> @github.com>
>

kuraakhilesh8230 commented:
> May I check it tomorrow please.
> 
> On Tue, 27 Sep, 2022, 11:33 pm Kura Akhilesh, ***@***.***>
> wrote:
> 
> > All checks failed (dco+lint)
> > Correct it asap
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/678#issuecomment-1259867956>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AVZCW3NNFGZIHTJZLFQDGXTWAMZITANCNFSM6AAAAAAQFA3GSQ>
> > .
> > You are receiving this because you modified the open/close state.Message
> > ID:
> > <scaleracademy/scaler-september-open-source-challenge/pull/678/c1259867956
> > @github.com>
> >
> 

Take your time and complete it by today

nayanapardhekar committed:
Delete HappyNumber.c

Signed-off-by: Nayana Pardhekar <nayanapardhekar@gmail.com>

nayanapardhekar committed:
Merge branch 'main' into nayanapardhekar-details

nayanapardhekar commented:
> > May I check it tomorrow please.
> > On Tue, 27 Sep, 2022, 11:33 pm Kura Akhilesh, _**@**_.***>
> > wrote:
> > > All checks failed (dco+lint)
> > > Correct it asap
> > > —
> > > Reply to this email directly, view it on GitHub
> > > [#678 (comment)](https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/678#issuecomment-1259867956),
> > > or unsubscribe
> > > https://github.com/notifications/unsubscribe-auth/AVZCW3NNFGZIHTJZLFQDGXTWAMZITANCNFSM6AAAAAAQFA3GSQ
> > > .
> > > You are receiving this because you modified the open/close state.Message
> > > ID:
> > > <[/pull/678](https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/678)/c1259867956
> > > @github.com>
> 
> Take your time and complete it by today

Done

kuraakhilesh8230 commented:
Congratulations you have successfully completed milestone 3 and good to go 👏

nayanapardhekar commented:
Thank you!

On Wed, 28 Sep, 2022, 7:43 am Kura Akhilesh, ***@***.***>
wrote:

> Congratulations you have successfully completed milestone 3 and good to go
> 👏
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/678#issuecomment-1260298977>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AVZCW3JCVKZR2I6UB6AMLA3WAOSUNANCNFSM6AAAAAAQFA3GSQ>
> .
> You are receiving this because you modified the open/close state.Message
> ID:
> <scaleracademy/scaler-september-open-source-challenge/pull/678/c1260298977
> @github.com>
>

nayanapardhekar committed:
Merge branch 'scaleracademy:main' into nayanapardhekar-details

nayanapardhekar committed:
Merge branch 'scaleracademy:main' into nayanapardhekar-details
